src/graph_notebook/magics/metadata.py

```diff
@@ -76,9 +76,9 @@ def set_gremlin_profile_metrics(gremlin_metadata: Metadata, profile_str: str) ->
     if querytime_regex:
         gremlin_metadata.set_metric_value('query_time', float(querytime_regex.group(1)))
     if predicates_regex:
-        gremlin_metadata.set_metric_value('predicates', int(predicates_regex.group(1)))
+        gremlin_metadata.set_metric_value('predicates', int(predicates_regex.group(1).replace(',', '')))
```

Comment:
Should we also include periods in here as that is common in most non-US countries to divide numbers?

Reply:
I'm unsure if the profile/explain text returned varies based on region. But that seems good to have just in case; I will add it